app/_data/docs_nav_gateway_3.3.x.yml

```diff
@@ -178,6 +178,8 @@ items:
             url: /production/monitoring/statsd
           - text: Datadog
             url: /production/monitoring/datadog
+          - text: Health Check
```

Comment:
We have too many things called "health check", this can be a source of confusion. Actually, with the existent health checks we already have some mix-up.

Reply:
How about `Readiness` or `Readiness Check`?